cli/cmd/root.go

```diff
@@ -126,31 +127,47 @@ func runCmd(ctx context.Context, ver cmdutil.Version) error {
 	rootCmd.PersistentFlags().StringVar(&ch.AdminTokenOverride, "api-token", "", "Token for authenticating with the cloud API")
 	rootCmd.Flags().BoolP("version", "v", false, "Show rill version") // Adds option to get version by passing --version or -v
 
-	// Add sub-commands
-	rootCmd.AddCommand(
+	// Command Groups
+
+	// Project commands
+	cmdutil.AddGroup(
+		rootCmd, "Project",
 		start.StartCmd(ch),
 		deploy.DeployCmd(ch),
-		env.EnvCmd(ch),
-		user.UserCmd(ch),
-		usergroup.UsergroupCmd(ch),
-		org.OrgCmd(ch),
 		project.ProjectCmd(ch),
 		publicurl.PublicURLCmd(ch),
 		service.ServiceCmd(ch),
+		runtime.RuntimeCmd(ch),
```

Comment:
The `RuntimeCmd` and `AdminCmd` commands are not user-facing (see [here](https://github.com/rilldata/rill/blob/07cb82a57feece481446b12d8b3dee197699811d/cli/cmd/runtime/runtime.go#L12) how they are hidden in release builds). We use them internally to start and scale those services separately in our Helm charts (all of Rill Cloud can be started with just this one binary, which simplifies things a lot). 

So I think they might better belong in the "Additional commands" group together with `sudo` and other internal commands (or if easy, perhaps have a dedicated `Internal` group, which is hidden in release builds).

Reply:
That makes sense, I like the idea of an `Internal` group.